frontend/app/src/components/CurrencyDropDown.vue

```diff
@@ -2,13 +2,13 @@
   <div>
     <v-menu transition="slide-y-transition" bottom offset-y>
       <template #activator="{ on }">
-        <v-btn
-          class="currency-dropdown secondary--text text--lighten-2"
-          icon
-          v-on="on"
+        <menu-tooltip-button
+          tooltip="Your main currency"
```

Comment:
Let's try to make these a bit uniform.

"Your main currency" --> "Main Currency" / "Profit Currency"

@LefterisJP perhaps we should use "profit currency" here as that's what we call it in the Usage Guide?

Notifications and Alerts are fine.

Reply:
Yes I think consistency is a good point here. It should be profit currency.

Reply:
👍